bin/test.py

```diff
@@ -281,6 +349,46 @@ def test_stack(session, config, stack_type):
             print 'Creating eb stack with enhanced db monitoring in: {}'.format(region)
             stack_ids.append(create_eb_stack(cfn_client, 'test-eb-enhanced-alarm-0', azs, 'dev', key_name, app_bucket_name, TEST_PYTHON_APP_KEY, 'python', 'mysql', True, True))
 
+        if stack_type == 'vpc-bastion-fargate' and region == 'us-east-1':
```

Comment:
Way too much repeated code here. We need to dry this out so that params are in a matrix and iterated over.

Reply:
Pushed to next set of changes for this script.